addons/binding/org.openhab.binding.wink/src/main/java/org/openhab/binding/wink/handler/BinarySwitchHandler.java

```diff
@@ -46,6 +46,11 @@ public void handleWinkCommand(ChannelUID channelUID, Command command) {
             }
         }
     }
+	
+	@Override
```

Comment:
this should likely be implemented in the BaseWinkDeviceHandler class rather than copy and pasted to each of the subclasses...overridden where necessary.